plugins/browser-plugin-enhanced-consent/README.md

```diff
@@ -0,0 +1,59 @@
+# Snowplow Enhanced Consent Plugin
+
+[![npm version][npm-image]][npm-url]
+[![License][license-image]](LICENSE)
+
+Browser Plugin to be used with `@snowplow/browser-tracker`.
+
+This plugin is the recommended way to track marketing consent events on your website. Usage and a complete setup journey is showcased on the [Consent Tracking for Marketing accelerator](https://docs.snowplow.io/accelerators/consent/).
+
+## Maintainer quick start
```

Comment:
This is a very tiny nitpick, but we usually put the maintainer steps behind the usage steps as that is the less frequent use case.